ogr/services/gitlab/project.py

```diff
@@ -241,6 +241,12 @@ def add_user(self, user: str, access_level: AccessLevel) -> None:
         except Exception as e:
             raise GitlabAPIException(f"User {user} already exists", e)
 
+    def request_access(self) -> None:
+        try:
+            self.gitlab_repo.accessrequests.create({})
+        except Exception as e:
```

Comment:
Can we use some Gitlab specific exception instead? (Don't we discuss this somewhere?..;)

Reply:
Yes on a different repo😅. Changing it.

Reply:
Done!